critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
```

Comment:
I don't recommend forking external libraries, but I can't say much else without knowing to what extent you've modified this. Can you please link to the original source code?

If we fork this, the file *must* include a copyright and license attached.